cli/dcoscli/metrics.py

```diff
@@ -0,0 +1,32 @@
+import contextlib
+from collections import OrderedDict
+
+from dcos import emitting, http, util
+from dcos.errors import DCOSException
+from dcoscli import tables
+
+logger = util.get_logger(__name__)
+emitter = emitting.FlatEmitter()
+
+
+def print_node_metrics_table(url):
+    """Retrieve, parse, and pretty-print the output from `dcos-metrics`' `node`
+    endpoint.
+
+    :param url: `dcos-metrics` `node` endpoint
+    :type url: str
+    """
+
+    with contextlib.closing(http.get(url)) as r:
+
+        if r.status_code != 200:
```

Comment:
This isn't an issue for `/node` metrics but other endpoints for metrics can return `204` (no content). Just a heads up when implementing against this API down the road.

Example: https://github.com/dcos/dcos-metrics/blob/master/producers/http/handlers.go#L102